meta-xt-dom0/recipes-kernel/linux/linux-generic-armv8/defconfig

```diff
@@ -2,15 +2,16 @@
 # Automatically generated file; DO NOT EDIT.
 # Linux/arm64 5.10.41 Kernel Configuration
 #
-CONFIG_CC_VERSION_TEXT="aarch64-poky-linux-gcc (GCC) 9.3.0"
+CONFIG_CC_VERSION_TEXT="aarch64-poky-linux-gcc (GCC) 11.3.0"
 CONFIG_CC_IS_GCC=y
-CONFIG_GCC_VERSION=90300
```

Comment:
Can we please limit changes only to functional ones?

Ideally we need to stop using `.config` as `defconfig` but this is completely another topic

Reply:
Can we please limit changes only to functional ones? - Yes, I will try

Reply:
"+"